### ISSUE TITLE
1.1.5 has breaking change - fails when no ignored devices defined

### ISSUE THREAD

**Issue by ikura** [asednev/homebridge-plugin-govee#59]
**Describe The Bug:**
Upgrading to 1.1.5 broke the plugin, causing a `TypeError`

**To Reproduce:**
Install 1.1.5
Configure with no ignored devices.

**Expected behavior:**
Expected the plugin to continue to work correctly,

**Logs:**

```
TypeError: Cannot read properties of undefined (reading 'includes')
    at GoveeHomebridgePlatform.goveeDiscoveredReading (/usr/local/lib/node_modules/homebridge-plugin-govee/dist/platform.js:76:32)
```

**Plugin Config:**

```json
{
            "name": "Govee",
            "batteryThreshold": 25,
            "debug": false,
            "humidityOffset": 0,
            "platform": "GoveeHomebridgePlugin"
        },
```

**Environment:**

* **Plugin Version**: 1.1.5
* **Homebridge Version**: 1.4.0
* **Node.js Version**: v16.14.1
* **NPM Version**: v8.5.0
* **Operating System**: Raspbian / hb-service

**Comment by ikura** [comment]
I found a workaround to just add random text to the 'ignored devices' part of the configuration

**Comment by MrFly72** [comment]
I will look at it on Monday.
Sorry for that, did not test it with no entries.
You workaround seems reasonable to have at least something in the list. As it is a simple string match that was added, it will just not find the string and add all other devices.

**Comment by MrFly72** [comment]
Already have a tested a fix (developed only my iPad) 😀
Will put it into code on Monday. List will now be checked if empty and check for ignored devices skipped.

**Comment by MrFly72** [comment]
Added a pull request for the project owner.
Hope i did it right, as GitHub is not really my "home" ;-)

**Comment by asednev** [comment]
@MrFly72 Thank you for a quick turn around on the fix.

I published version 1.1.6 that fixes the issue. Sorry for the delay, it took me some time to fix my setup to test everything before publishing.
